app/gui/controller/engine-protocol/src/language_server/tests.rs

```diff
@@ -479,6 +480,7 @@ fn test_execution_context() {
     };
     let positional_arguments_expressions = vec!["foo"].iter().map(|x| x.to_string()).collect();
     let visualisation_config = VisualisationConfiguration {
+        visualisation_module: "local.Widgets.Main",
```

Comment:
This string is used in so many places, it feels like it should be a constant.

btw. what does it point to? I'm not aware of a library under this path.

Reply:
The string points to a file on my local disk. The string may be used few times, but only in tests, not in production code. The `visualisation_module` argument needs to be set to something. It (very likely) doesn't matter what the value is in the existing tests.

Reply:
I was analysing what `local.Widgets.Main` could be pointing to. Couldn't figure it out and was confused. Now I know that the file was just not existing in the repo.

I think it will be clearer if the name reflects this, maybe let's call it then `local.Nonexistent_Arbitrary_Project.Main`?